x-pack/test/security_solution_cypress/es_archives/unmapped_fields/mappings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "index",
+  "value": {
+    "aliases": {
+      "mydata-7.12.0": {
+        "is_write_index": true
```

Comment:
Do you need/want `is_write_index: true` here? Isn't that more for ILM stuff and might cause issues?

I would remove this if possible. I don't think we use this alias.